holoviews/plotting/bokeh/links.py

```diff
@@ -138,10 +138,16 @@ class RangeToolLinkCallback(LinkCallback):
     def __init__(self, root_model, link, source_plot, target_plot):
         toolbars = list(root_model.select({'type': Toolbar}))
         axes = {}
-        if 'x' in link.axes:
-            axes['x_range'] = target_plot.handles['x_range']
-        if 'y' in link.axes:
-            axes['y_range'] = target_plot.handles['y_range']
+        for axis in ['x', 'y']:
+            if axis in link.axes:
+                axes[f'{axis}_range'] = target_plot.handles[f'{axis}_range']
+                bounds = getattr(link, f'bounds{axis}', None)
+                if bounds is not None:
+                    start, end = bounds
+                    if start is not None:
+                        axes[f'{axis}_range'].start = start
+                    if end is not None:
+                        axes[f'{axis}_range'].end = end
```

Comment:
I wonder if we can reduce the indentation like this?
```python
for axis in ['x', 'y']:
    if axis not in link.axes:
        continue

    axes[f'{axis}_range'] = target_plot.handles[f'{axis}_range']
    bounds = getattr(link, f'bounds{axis}', None)
    if bounds is None:
        continue

    start, end = bounds
    if start is not None:
        axes[f'{axis}_range'].start = start
    if end is not None:
        axes[f'{axis}_range'].end = end
```


(Relevant video explanation: https://www.youtube.com/watch?v=CFRhGnuXG-4)

Reply:
thanks! this is a really helpful concept and video :)